src/Config/ObjectModel/McpDmlToolsOptions.cs

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json.Serialization;
+
+namespace Azure.DataApiBuilder.Config.ObjectModel;
+
+/// <summary>
+/// DML Tools found in global MCP configuration.
```

Comment:
nit: explain in the summary that these options are useful for interacting with the CLI